server/src/pages/shot/time-diff.js

```diff
@@ -24,6 +24,12 @@ exports.TimeDiff = class TimeDiff extends React.Component {
     }
   }
 
```

Comment:
I'm also wondering if fluent-react supports the advanced options shown here: http://l20n.org/learn/advanced-selectors